test/clientTest.js

```diff
@@ -25,11 +24,9 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
     minMem: 1024,
     maxMem: 1024
   })
-  wrap.on('line', function (line) {
-    console.log(line)
-  })
+  // wrap.on('line', console.log);
```

Comment:
Why did you remove this ? This is very useful to know what didn't work in a test.

Reply:
Tests shouldn't log useless data, they should provide a clean output so the user can see what's going on. It's not really useful to have the minecraft server log. Don't really know why you would want that

Reply:
This is the most useful data in the client tests. The client tests are testing against the vanilla server so we want to know everything we can.

In practice it was possible to catch several problems using the vanilla server logs (recently that a test was stuck even though the vanilla server correctly saw the player log in)